generators/entity-client/templates/react/src/main/webapp/app/entities/_entity.tsx

```diff
@@ -61,7 +61,7 @@ export interface I<%= entityReactName %>State {
 }
 <%_ } _%>
 
-export class <%= entityReactName %> extends React.Component<I<%= entityReactName %>Props, I<%= entityReactName %>State> {
+export class <%= entityReactName %> extends React.Component<I<%= entityReactName %>Props, <% if (searchEngine === 'elasticsearch') { %>I<%= entityReactName %>State<% } else { %>undefined<% } %>> {
```

Comment:
can you change this to
`<I<%= entityReactName %>Props<% if (searchEngine === 'elasticsearch') { %>, I<%= entityReactName %>State<% } %>> {`